genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :id, :items
+
+  def initialize(name, id)
+    @id = id
+    @name = name
+    @items = items[]
+  end
+
+  def add_item(item)
+    @items << item
+    item.genre = self
```

Comment:
I don't think that genre is a property of an item but it's a method inside it so please write a correct syntax and pass self notation to it as an argument :ok_hand: